app/classifier/tasks/survey/summary.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import Utility from './utility';
+
+class SurveySummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expanded: props.expanded
```

Comment:
I think passing props to state isn't an anti-pattern, if you were just seeding data that doesn't need to be updated during the lifecycle. Is that the case?

Reply:
Could be. I think this is just copied straight across from the original code.

Reply:
Expanded state is changed by clicking the buttons to open/close the summary. Is that what you were asking about?

Reply:
As far as I can see, the expand button works properly, so this isn't blocking. However, for being extra cautious, could we just have `false` as a default in the constructor, instead of feeding the props?

Reply:
But the tests pass in `expanded=true` since you can render with the summary initially open, or am I misunderstanding?

Reply:
I think I'm just being rubbish at being clear. My thought: if `props.expanded` is never going to be `true` on first render, why don't just assign `false` to the local initial state and avoid a possible antipattern?

Reply:
And my reply was that `expanded` can be true on first render, hence there's a test for it. I feel like I'm missing something obvious here.